task/buildah/0.4/buildah.yaml

```diff
@@ -535,6 +528,10 @@ spec:
         fi
       done
 
+      image_name=$(echo "${IMAGE##*/}" | tr ':' '-')
+      buildah push "$IMAGE" oci-archive:"./$image_name.tar"
+      echo "$(pwd)/$image_name.tar" > /shared/container_path
```

Comment:
What is the `$PWD` at this point? If I'm not mistaken, it's the root of the git repo, so we're writing the oci-archive into the root of the git repo?

Could we avoid that and instead write it to e.g. `/shared/image-archive.tar`?